tests/unit/test_coco_validation.py

```
@@ -149,6 +150,76 @@ def test_coco_evaluator_integration(tmp_path):
+    validator.config = SimpleNamespace(
+        data=str(yaml_path),
+        data_dir=None,
+        split="val",
+        allow_download_scripts=False,
+        imgsz=64,
+        batch_size=1,
+        num_workers=0,
+    )
+    validator.model = SimpleNamespace(
+        _get_input_size=lambda: 64,
+        _get_val_preprocessor=lambda img_size: None,
```

Ruff (0.15.15)

[warning] Unused lambda argument: `img_size`

(ARG005)